src/main/java/de/ustutt/iaas/cc/core/RemoteTextProcessorMultiLeastConnection.java

```diff
@@ -0,0 +1,79 @@
+package de.ustutt.iaas.cc.core;
+
+import java.util.Collections;
+import java.util.List;
+import java.util.PriorityQueue;
+import java.util.concurrent.CopyOnWriteArrayList;
+import java.util.concurrent.CopyOnWriteArraySet;
+
+import javax.ws.rs.client.Client;
+import javax.ws.rs.client.Entity;
+import javax.ws.rs.client.WebTarget;
+import javax.ws.rs.core.MediaType;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class RemoteTextProcessorMultiLeastConnection implements ITextProcessor {
```

Comment:
Some comments describing your general idea and solution approach would be helpful here.